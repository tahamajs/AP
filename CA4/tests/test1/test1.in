report_salaries
report_employee_salary 2
report_employee_salary 3
add_working_hours 1 2 20 21
add_working_hours 4 1 14 15
report_salaries
report_employee_salary 4
delete_working_hours 3 6
report_salaries
