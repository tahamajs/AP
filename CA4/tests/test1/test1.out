ID: 1
Name: AzadehZahedi
Total Working Hours: 200
Total Earning: 21250
---
ID: 2
Name: GolnazEmadi
Total Working Hours: 217
Total Earning: 53761
---
ID: 3
Name: ParvinKarami
Total Working Hours: 228
Total Earning: 54857
---
ID: 4
Name: MehranAmiri
Total Working Hours: 226
Total Earning: 46749
---
ID: 5
Name: BehzadAbbasi
Total Working Hours: 175
Total Earning: 28080
---
ID: 2
Name: GolnazEmadi
Age: 39
Level: team_lead
Team ID: 1
Total Working Hours: 217
Absent Days: 2
Salary: 79060
Bonus: 0
Tax: 25299
Total Earning: 53761
ID: 3
Name: ParvinKarami
Age: 45
Level: team_lead
Team ID: 2
Total Working Hours: 228
Absent Days: 2
Salary: 80672
Bonus: 0
Tax: 25815
Total Earning: 54857
OK
OK
ID: 1
Name: AzadehZahedi
Total Working Hours: 201
Total Earning: 21301
---
ID: 2
Name: GolnazEmadi
Total Working Hours: 217
Total Earning: 53761
---
ID: 3
Name: ParvinKarami
Total Working Hours: 228
Total Earning: 54857
---
ID: 4
Name: MehranAmiri
Total Working Hours: 227
Total Earning: 46833
---
ID: 5
Name: BehzadAbbasi
Total Working Hours: 175
Total Earning: 28080
---
ID: 4
Name: MehranAmiri
Age: 32
Level: expert
Team ID: 1
Total Working Hours: 227
Absent Days: 0
Salary: 62444
Bonus: 0
Tax: 15611
Total Earning: 46833
OK
ID: 1
Name: AzadehZahedi
Total Working Hours: 201
Total Earning: 21301
---
ID: 2
Name: GolnazEmadi
Total Working Hours: 217
Total Earning: 53761
---
ID: 3
Name: ParvinKarami
Total Working Hours: 219
Total Earning: 54006
---
ID: 4
Name: MehranAmiri
Total Working Hours: 227
Total Earning: 46833
---
ID: 5
Name: BehzadAbbasi
Total Working Hours: 175
Total Earning: 28080
---
